connector_jira/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 
 {
     'name': 'JIRA Connector',
-    'version': '12.0.1.2.0',
+    'version': '12.0.1.3.0',
```

Comment:
FTR this is not needed as we use the bot to merge ;)

Reply:
:) Old habit :)